src/app/routes/index.js

```diff
@@ -9,7 +9,7 @@ const routes = [
     getInitialData,
   },
   {
-    path: '/:service/articles/amp/:id',
+    path: '/:service/articles/:id(.amp)?',
```

Comment:
As discussed offline, there can be single route that has a path of `/:service(news|persian)/articles/:id(c[a-zA-Z0-9]{10}o):amp(.amp)?`.

This has the following benefits:
1. Validation of the service in the route, so don't have to do custom validation of the `service` param in `getInitialData`
2. Validation of the article ID in the route, so don't have to custom validation of the `id` param in `getInitialData`
3. A single route, so don't need a separate route for amp, and it will come through as a param called `amp`.  As it is an optional regex group, it will be `undefined` if not-AMP or pass the `.amp` string if amp.

This means that [all validation](https://github.com/bbc/simorgh/blob/d6b7b9ab6f38829386641fc1f7b316032018ae1d/src/app/routes/getInitialData/index.js#L31-L35) can be removed from `getInitialData`.

Reply:
As discussed, this is way closer to what I wanted, just got lost a bit along the way.

Quick observation from implementing it: while it works on the [route-tester linked from the path-to-regexp repo](http://forbeslindesay.github.io/express-route-tester/), in practice with react-router, this is always sending `{ service: 'news', id: 'c0000000025o.amp' }`. The `amp` regex group doesn't seem to come through. I'll keep looking.

Reply:
Using ^ and $ to anchor the id regex to the start and end of the actual id might help:

 `:id(^c[a-zA-Z0-9]{10}o$):amp(.amp)?`

Reply:
Have you removed the first route?

Reply:
Oh my goodness, that feels so obvious 🤦🏻‍♂️ Pre-coffee coding, bad idea. Sorry!

Reply:
I didn't remove the first route when I was testing that path last night for ages... which is why I knew what it was!

Reply:
I've made these changes, one downside I'd expect would be for the test coverage to go down as it removed some tests, I suspect these will have to be replaced by more comprehensive e2e tests.